content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -226,6 +226,64 @@ As a client, you can request `BOOKMARK` events by setting the
 assume bookmarks are returned at any specific interval, nor can clients assume that
 the API server will send any `BOOKMARK` event even when requested.
 
+## Streaming lists {#watch-bookmarks}
```

Comment:
This will shadow the `## Watch bookmarks` heading earlier.
I have a suspicion this was my suggestion; anyway, I suggest:


```suggestion
### Streaming lists {#watch-bookmark-initial-events}
```

The new text becomes part of “Efficient detection of changes”.

---

If we need to reframe that 2nd-level heading, perhaps to be “Data streams”, “Streamed collection data” or something else more generic, then we can do so.

Is there a case where a client does a **watch** with `sendInitialEvents` other than to end up running the watch? (Is there a reasonable case where client might want to abort the watch as soon as it sees the first `BOOKMARK` event?)

Reply:
That wasn't your suggestion - it was @tallclair 's one.

But I messed it up - it should be for "watch bookmarks" section. Should be fixed now.


> Is there a reasonable case where client might want to abort the watch as soon as it sees the first BOOKMARK event?

There might be a reasonable usecase where we actually want to stream the data without storing everything in memory. So yes - there are usecases (although we don't want to endorse them yet - probably in the future).